views/partials/dashboards/remove-enroll.hbs

```diff
@@ -21,9 +21,12 @@
     <div class="flx">
       <form id="remove-enroll-form" action="/user/remove-enroll-submit">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
-        <p>
-          <a href="#">{{getRemoveString "remove-enroll-terms-agreement"}}</a>
-        </p>
+        <div class="flx">
+          <input id="remove-enroll-terms" class="remove-enroll-terms" type="checkbox" name="terms" required>
+          <label for="remove-enroll-terms">{{getRemoveString "remove-enroll-terms-agreement"}} 
+            <a href="/removal-pilot-terms">{{getRemoveString "remove-enroll-terms-link"}}</a> &
+            <a href="/removal-pilot-privacy">{{getRemoveString "remove-enroll-privacy-link"}}</a></label>
```

Comment:
Q: Should the links be in the `<label>...</label>` as well? or just the localized `terms-agreement` text?